packages/datadog_flutter_plugin/doc/rum/advanced_configuration.md

```diff
@@ -121,6 +121,102 @@ For example:
+|                  | `errorEvent.view.referrer`        | Referrer of the view linked to this action.   |
+|                  | `errorEvent.view.url`             | URL of the view linked to this error.         |
+| RumResourceEvent | `resourceEvent.resource.url`      | URL of the resource.                          |
+|                  | `resourceEvent.view.name`         | Name the view linked to this action.          |
+|                  | `resourceEvent.view.referrer`     | Referrer of the view linked to this action.   |
+|                  | `resourceEvent.view.url`          | URL of the view linked to this resource.      |
+
+## Set tracking consent (GDPR compliance)
+
+To be compliant with the GDPR regulation, the Flutter RUM SDK requires the tracking consent value at initialization.
+
+The `trackingConsent` setting can be one of the following values:
+
+1. `TrackingConsent.pending`: The Flutter RUM SDK starts collecting and batching the data but does not send it to Datadog. The Flutter RUM SDK waits for the new tracking consent value to decide what to do with the batched data.
+2. `TrackingConsent.granted`: The Flutter RUM SDK starts collecting the data and sends it to Datadog.
+3. `TrackingConsent.notGranted`: The Flutter RUM SDK does not collect any data. No logs, traces, or RUM events are sent to Datadog.
+
+To change the tracking consent value after the Flutter RUM SDK is initialized, use the `DatadogSdk.setTrackingConsent` API call. The Flutter RUM SDK changes its behavior according to the new value. 
+
+For example, if the current tracking consent is `TrackingConsent.pending`:
+
+- If you change the value to `TrackingConsent.granted`, the Flutter RUM SDK sends all current and future data to Datadog;
+- If you change the value to `TrackingConsent.notGranted`, the Flutter RUM SDK wipes all current data and does not collect future data.
+
+## Sample RUM sessions
+
+To control the data your application sends to Datadog RUM, you can specify a sampling rate for RUM sessions while [initializing the Flutter RUM SDK][2] as a percentage between 0 and 100.
+
+For example, to only keep 50% of sessions use:
+
+```dart
+final config = DdSdkConfiguration(
+    // other configuration...
+    rumConfiguration: RumConfiguration(
+        applicationId: '<YOUR_APPLICATION_ID>',
+        sessionSamplingRate: 50.0,
+    ),
+);
+```
+
+## Sending data when device is offline
+
+RUM ensures availability of data when your user device is offline. In cases of low-network areas, or when the device battery is too low, all the RUM events are first stored on the local device in batches. They are sent as soon as the network is available, and the battery is high enough to ensure the Flutter RUM SDK does not impact the end user's experience. If the network is not available while your application is in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
```

Comment:
```suggestion
RUM ensures availability of data when your user device is offline. In cases of low-network areas, or when the device battery is too low, all RUM events are first stored on the local device in batches. They are sent as soon as the network is available, and the battery is high enough to ensure the Flutter RUM SDK does not impact the end user's experience. If the network is not available with your application running in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
```